src/shared/state/Applet/Applet.reducer.ts

```diff
@@ -13,4 +13,12 @@ export const extraReducers = (builder: ActionReducerMapBuilder<AppletSchema>): v
   createAppletPendingData({ builder, thunk: getApplet, key: 'applet' });
   createAppletFulfilledData({ builder, thunk: getApplet, key: 'applet' });
   createAppletRejectedData({ builder, thunk: getApplet, key: 'applet' });
+
+  createAppletPendingData({ builder, thunk: createApplet, key: 'applet' });
```

Comment:
I thin it's better to use common: 
appletPendingData, appletFullfilledData, appletRejectedData